### DISCUSSION TITLE
NFC Plugin

### DISCUSSION THREAD

**Discussion by Logan-Fidelitab** [capawesome-team/capacitor-plugins#166] [Q&A] [answered]
Hi, I'm interested by the NFC Capacitor plugin but does it allow to send NDEF message between two mobile devices ? Such as between a smartphone and a tablet. I raed the doc but it's not mentioned.

Thanks

**Comment by robingenz** [comment, marked as answer]
Yes, this is not yet supported. There is already an issue for that: https://github.com/capawesome-team/capacitor-plugins/issues/155

**Reply by Logan-Fidelitab** [reply]
Ok thanks, will this feature be implemented in the future ?

**Reply by robingenz** [reply]
Yes, but there is no ETA yet.
